### ISSUE TITLE
Error uploading File

### ISSUE THREAD

**Issue by RogersNtr** [Jefiozie/ngx-aws-deploy#154]
**Describe the bug**
Hi, I don't know if this is a bug, but when trying to deploy, I have a message: 
`Error uploading file: undefined` when using `ng deploy`. The message is output many time, until `Finished uploading files` is reached.

I double check the AWS user credentials, and they are all fines. So I'm a little bit confused. Any hint ?

**Comment by Jefiozie** [comment]
Hi @RogersNtr, what I can tell you is that I've seen this message appear when the credentials are not correct or that for some reason you can't connect to the AWS S3 bucket. Could you share maybe a small repo where you can reproduce the issue? Or at least the setup without the credentials etc. of course.
Maybe that can help me in the correct direction of finding the problem together with you.

**Comment by Jefiozie** [comment]
I'm changing the error log in #173 so that the error message is coming straight from S3. I'm curious if you have found a issue? If you resolved it would you mind and a comment to the issue so that other people can see your solution? 
Thanks.

**Comment by baliganikhil** [comment]
I had this issue as well. The credentials are working fine when I use the credentials with aws-cli.

I was able to get it resolved. Here is the problem - Perhaps this can help you.

I didn't want to use the credentials by putting them in the `angular.json` file. Immediately after installing the package, I was asked to provide the credentials. I added a dummy value for access key and secret.

I set all the environment variables and tried to use it. However, it appears that environment variables are not getting picked up. I added the credentials in the angular.json file and the files began to upload (mostly - I got only one or two errors, but generally almost everything got uploaded)
